src/main/java/org/skriptlang/skript/bukkit/itemcomponents/equippable/elements/EffEquipCompDispensable.java

```diff
@@ -0,0 +1,56 @@
+package org.skriptlang.skript.bukkit.itemcomponents.equippable.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableExperiment;
+import org.skriptlang.skript.bukkit.itemcomponents.equippable.EquippableWrapper;
+
+@Name("Equippable Component - Dispense")
+@Description("If the item can be dispensed by a dispenser. "
+	+ "Note that equippable component elements are experimental making them subject to change and may not work as intended.")
+@Example("allow {_item} to be dispensed")
+@Example("""
+	set {_component} to the equippable component of {_item}
+	prevent {_component} from being dispensed
+	""")
+@RequiredPlugins("Minecraft 1.21.2+")
+@Since("INSERT VERSION")
+public class EffEquipCompDispensable extends Effect implements EquippableExperiment {
+
+	static {
+		Skript.registerEffect(EffEquipCompDispensable.class,
+			"allow %equippablecomponents% to be dispensed",
+			"prevent %equippablecomponents% from (being dispensed|dispensing)"
```

Comment:
```suggestion
			"prevent %equippablecomponents% from being dispensed"
```
the item doesn't do the dispensing

Reply:
not resolved. i'd also consider allowing `block`: `(prevent|block)`